content/hardware/02.hero/boards/uno-r4-minima/tutorials/opamp/opamp.md

```diff
@@ -0,0 +1,107 @@
+---
+title: 'Arduino UNO R4 Minima OPAMP'
+description: 'Learn how to use the built-in Operational Amplifier in the UNO R4 Minima'
+tags:
+  - OPAMP
+author: 'Maximilian Gerhardt, Hannes Siebeneicher'
+hardware:
+  - hardware/02.hero/boards/uno-r4-minima
+software:
+  - web-editor
+  - ide-v1
+  - ide-v2
+---
+
+In this tutorial, you will learn how to use the built-in operational amplifier (OPAMP) featured on the [Arduino UNO R4 Minima](https://store.arduino.cc/products/uno-r4-minima). Operational amplifiers are very versatile and can be used to e.g. mirror an electrical signal or amplify it. 
+
+## Goals
+
+In this article, you will learn:
+- about the OPAMP feature onboard the UNO R4 Minima,
+- about the basics of the OPAMP library, 
+- how to mirror a voltage signal,
+- how to amplify a voltage signal.
+
+## Hardware & Software Needed
+  To follow along with this article, you will need the following hardware: 
+  
+  - [Arduino UNO R4 Minima](https://store.arduino.cc/uno-r4-minima)
+  - resistors 
+  - Jumper wires
+  
+  
+## Operational Amplifier (OPAMP)
+
+An OPAMP is a versatile and widely used electronic component that belongs to the class of analog integrated circuits. Its primary function is to amplify voltage signals but they are very versatile and can be used to:
+
+- mirror an input voltage to its output,
+- amplify a small analog voltage to its output pin, output voltage range from 0 to ~4.7 V,
```

Comment:
```suggestion
- amplify a small analog voltage to its output pin, the UNO R4's output voltage range is 0 to ~4.7 V,
```

Reply:
Makes it clear in the section about generic OPAMPS that this note is about the UNO R4 OPAMP specifically :-)